jekyll/assets/js/highlightJSBadge.js

```diff
@@ -10,29 +10,34 @@ window.onload = () => {
          var options = {
             // the selector for the badge template
             templateSelector: "#CodeBadgeTemplate",
-      
+
             // base content CSS selector that is searched for snippets
             contentSelector: "body",
-      
+
             // Delay in ms used for `setTimeout` before badging is applied
             // Use if you need to time highlighting and badge application
             // since the badges need to be applied afterwards.
             // 0 - direct execution (ie. you handle timing
             loadDelay: 0,
-      
+
             // CSS class(es) used to render the copy icon.
-            copyIconClass: "fa fa-copy",     
+            copyIconClass: "fa fa-copy",
             // optional content for icons class (<i class="fa fa-copy"></i> or <i class="material-icons">file_copy</i>)
             copyIconContent: "",
-      
+
             // CSS class(es) used to render the done icon.
             checkIconClass: "fa fa-check text-success",
             checkIconContent: "",
-      
+
             // function called before code is placed on clipboard that allows you inspect and modify
             // the text that goes onto the clipboard. Passes text and code root element (hljs).
             // Example:  function(text, codeElement) { return text + " $$$"; }
-            onBeforeCodeCopied: null        
+            onBeforeCodeCopied: function(text) {
+              window.AnalyticsClient.trackAction('copy-code-clicked', {
+                page: location.pathname
```

Comment:
Should this be prefixed with `dd` ? or `docs-copy-code-clicked` ?

Reply:
We don't prefix our events with `dd` (only experiments key) but I like the suggestion of `docs-copy-code-clicked` 🙌

Reply:
I think track actions, we typically don't prefix with dd. but it would be nice to add something like { isExperiment: true } in the options for experiments that touch exists trackActions.

since this one is new and does not occur in control, it would be superfluous.